app/views/layouts/_navbar.html.erb

```diff
@@ -3,8 +3,7 @@
   <nav class="navbar navbar-expand-lg navbar-light bg-white fixed-top">
     <div class="container-fluid">
     <a class="navbar-brand" href="#">
-      <img src="" alt="" width="30" height="24" class="d-inline-block align-text-top">
-      Recipe App
+      <%= image_tag 'hklogocircle.png', size: "75x75", class: "d-inline-block align-text-top rounded-circle"%>
```

Comment:
This fine but it feels like it should be in its own PR. Just for future reference.